about.html

```diff
@@ -0,0 +1,155 @@
+                world. <br> Our team has come together t make it possible for everyone to become a software developer with a curriculum containing all essential and important courses to succeed in your career.
+                We create simplified and interactive learning experiences. <br>
+                Learning web development should be easy to understand and available for everyone, everywhere!
+                We believe, learning is the source of human progress.It has the power to transform our lives, for ourselves, for our families, for our communities.</p>
+            <p class="contact_us">Please contact us per Email for any further questions</p>
+            <p class="credit">@ Web Course</p>
+            <p class="email">webcourse@dev.com</p>
+        </div>
+    </section>
+    <section class="logo_design d_flex_column">
+        <div class="section_title d_flex_column">
+            <h2 class="title title2lines">Web Development Course</h2>
+            <hr>
+        </div>
+        <p class="logo_story">The logo of Web Development Course was created James Smith who is a student who join this
+            courses to become a software developer.</p>
+        <div class="logo_container logo_section2 d_flex_row">
+            <img src="./images/graduation-cap2.png" alt="graduation cap">
+            <div class="logo_name d_flex_column">
+                <h2>Web <span> Dev</span></h2>
+                <h3>Course</h3>
+            </div>
+        </div>
+    </section>
+    <section class="future_classes d_flex_column">
+        <div class="section_title d_flex_column">
+            <h2 class="title">Next Future Classes</h2>
+            <hr>
+        </div>
+        <p class="schedules">Here is the program and the start date for the next classes.</p>
+        <div class="cards_date">
+            <div class="class_date class1">
+                <h3 class="date">January 13th 2022</h3>
+                <a class="start_now" href="#">Start Now</a>
+            </div>
+            <div class="class_date class2">
+                <h3 class="date"> March 4th 2022</h3>
+                <a class="start_now" href="#">Start Now</a>
+            </div>
+        </div>
+    </section>
+    <Footer>
+        <div class="partners_section">
+            <div class="section_title d_flex_column">
+                <h2 class="title footer_title">Partner</h2>
+                <hr>
+            </div>
+            <ul class="partners d_flex_row">
+                <li><img class="partner_img" src="./images/google.png" alt="google"></li>
+                <li><img class="partner_img" src="./images/ted.png" alt="ted"></li>
+                <li><img class="partner_img" src="./images/blogger.png" alt="blogger"></li>
+                <li><img class="partner_img" src="./images/windows-logo.png" alt="windows-logo"></li>
+                <li><img class="partner_img" src="./images/github.png" alt="github"></li>
```

Comment:
- [ ] [OPTIONAL] I admire the effort you put in working on this part of your project. I suggest we try to reduce the size of the images.